### ISSUE TITLE
Is it possible to train the model using more than 4 GPUs and also with distributed computing ?

### ISSUE THREAD

**Issue by surajitkundu29** [AlexeyAB/darknet#4483]
For Ex. I have two GPU enabled server and bot having 4 GPUs, How my training can be distributed over these two server ?

**Comment by AlexeyAB** [comment]
No. There isn't implemented the training of one model on several servers.

* **without** synchronized-batch-normalization **it will hurt accuracy** - training on more than 4 x GPUs

* **with** synchronized-batch-normalization **it will hurt training speed** - training on several servers (but you will be able to use large mini_batch_size that will increase accuracy) - instead for this I suggest to use GPU-processing on CPU-RAM: https://github.com/AlexeyAB/darknet/issues/4386
